eth/tools/fixtures/helpers.py

```diff
@@ -156,7 +161,7 @@ def chain_vm_configuration(fixture: Dict[str, Any]) -> Iterable[Tuple[int, Type[
             (0, SpuriousDragonVM),
             (5, ByzantiumVM),
         )
-    elif network == 'ByzantiumToConstantinopleAt5':
+    elif network == 'ByzantiumToConstantinopleFixAt5':
         return (
             (0, ByzantiumVM),
             (5, ConstantinopleVM),
```

Comment:
Since the network has `ConstantinopleFix` shouldn't this be replaced by `(5, PetersburgVM)`?

Reply:
Right!

I'd guess this still passes, because the only upstream test that uses this doesn't have anything `Petersburg`-specific.

Reply:
This is my fault because this particular line was changed by me. I didn't pay much attention to this because the test was passing until @cburgdorf pointed it out [here](https://github.com/ethereum/py-evm/pull/1852#discussion_r330404283).

Reply:
Opened [an issue upstream](https://github.com/ethereum/tests/issues/644) anyway, so it doesn't get lost.